activesupport/test/message_encryptor_test.rb

```diff
@@ -119,7 +119,7 @@ def assert_not_decrypted(value)
     end
 
     def assert_not_verified(value)
-      assert_raise(ActiveSupport::MessageVerifier::InvalidSignature) do
+      assert_raise(ActiveSupport::MessageEncryptor::InvalidMessage) do
```

Comment:
Previously the setup method used  **aes-256-cbc**  cipher while writing this test and had made use of Message Verifier. However, after playing around with the tests, I realized this test was failing because ActiveSupport::MessageVerifier::InvalidSignature was raised by message verifier, and the new default cipher - **aes-256-gcm** used NullVerifier.